internal/clients/http/sources/sources_client_test.go

```diff
@@ -62,30 +62,6 @@ func TestSourcesClient_GetAuthentication(t *testing.T) {
 
 		assert.Equal(t, "1a2b3c4d-5e6f-7a8b-9c8d-7e8f9a8b7c6d", authentication.Payload)
 	})
-
-	t.Run("source with multiple apps", func(t *testing.T) {
```

Comment:
Why dont we want to cover this case? it is common for the Superkey method for AWS to end up with state like this, so I wonder why we are removing coverage for it? :thinking:

Reply:
ah, right we are filtering it all out on the sources side, we won't get this result here, makes sense :+1: IGNORE me :)